src/js/service/PiskelApiService.js

```diff
@@ -178,6 +185,41 @@
   };
 
   /**
+   * @param {!string} uri
```

Comment:
Might be worth a function level comment saying what it means to load frames? (i.e. does it essentially just mean to download the frames?)